src/Quiz.ts

```diff
@@ -16,6 +16,8 @@ const getDrawnPath = (userStroke: UserStroke) => ({
 });
 
 export default class Quiz {
+  // Armazena o resultado de cada traço desenhado
```

Comment:
Please don't leave code comments in Portuguese. All code comments should be in English.